lib/merchant_repository.rb

```diff
@@ -0,0 +1,41 @@
+class MerchantRepository
+  # This is the MerchantRepository Class
+  # frozen_string_literal: true.
+  attr_reader :merchants
+
+  def initialize
+    @merchants = []
```

Comment:
Brandon and I are immediately taking the data and creating merchants/items/etc in our Repos initialize methods. We can revisit this, both work.